packages/riipen-ui/src/components/VerticalProgressBarItem.jsx

```diff
@@ -101,43 +96,50 @@ const VerticalProgressBarItem = ({ progress }) => {
           display: flex;
           height: 24px;
           justify-content: center;
-          margin: ${theme.spacing(2)}px 0 ${theme.spacing(7)}px -23px;
-          padding: 7px;
```

Comment:
Make icon circle a bit smaller, update margins to accommodate (and also put in margin-bottom, margin-left etc. to make it easier to read, can put back into a one-liner if desired!)